cmd/bedPeOverlap/bedPeOverlap.go

```diff
@@ -125,6 +127,51 @@ func SelectIsBed(bedSelectFile string, bedpeInFile string, overlapThreshold floa
 	exception.PanicOnErr(err)
 }
 
+func SelectIsBedBoth(bedSelectFile string, bedpeInFile string, overlapThreshold float64, contactOutFile string) {
+	var selectIntervals = make([]interval.Interval, 0)
+	var currOverlaps []interval.Interval
+	var bothOverlap []interval.Interval
+	var err error
+	var found bool
+
+	selectRecords := bed.Read(bedSelectFile)
+	inBedPe := bedpe.Read(bedpeInFile)
+	out := fileio.EasyCreate(contactOutFile)
+
+	for _, i := range selectRecords {
+		selectIntervals = append(selectIntervals, i)
+	}
+	selectTree := interval.BuildTree(selectIntervals)
+
+	for _, i := range inBedPe {
+		currOverlaps = interval.Query(selectTree, i.A, "any")
```

Comment:
Optional, but it might be more descriptive to rename currOverlaps -> aOverlaps and bothOverlap -> bOverlaps